zerver/worker/queue_processors.py

```diff
@@ -680,22 +681,33 @@ def failure_processor(event: Dict[str, Any]) -> None:
             start = time.time()
             realm = Realm.objects.get(id=event['realm_id'])
             output_dir = tempfile.mkdtemp(prefix="zulip-export-")
+            export_event = RealmAuditLog.objects.get(id=event['id'])
+            user_profile = get_user_profile_by_id(event['user_profile_id'])
+
+            try:
+                public_url = export_realm_wrapper(realm=realm, output_dir=output_dir,
+                                                  threads=6, upload=True, public_only=True,
+                                                  delete_after_upload=True)
+            except Exception:
+                export_event.extra_data = ujson.dumps(dict(
+                    failed_timestamp=timezone_now().timestamp()
+                ))
+                export_event.save(update_fields=['extra_data'])
+                logging.info("Data export for %s failed after %s" % (
```

Comment:
This should be `logging.error` -- failures here aren't supposed to happen and should be reported as errors.